### PR TITLE
twonkie/introduction.md: process improvements

### PR DESCRIPTION
**Pull request by Psotas**

### CODE REVIEW

File: docs/transparent-validation/twonkie/introduction.md [hunk 1]
```diff
@@ -20,56 +20,70 @@ Links:
...
+    > start after ~5 minutes.
+
     ```bash
-    $ sudo dfu-util -a 0 -s 0x08000000 -D ~/Downloads/twonkiev2-20230611.bin
+    sudo dfu-util -a 0 -s 0x08000000 -D ~/Downloads/twonkiev2-20230611.bin
     ```
 
-    > The command may take a long time to complete. Do not be alarmed if progress
-    > appears to stop.
+    > If u see ```DFU state(10) = dfuERROR, status(10) = Device's firmware is```
```
Comment (mkopec) [inline docs/transparent-validation/twonkie/introduction.md:76]:
```suggestion
    > If you see ```DFU state(10) = dfuERROR, status(10) = Device's firmware is```
```

File: docs/transparent-validation/twonkie/introduction.md [hunk 2]
```diff
@@ -20,56 +20,70 @@ Links:
...
+    ```corrupt. It cannot return to run-time (non-DFU) operations```
+    > this is issue with dfu-util, Unfortunately, it's normal, flashing is
```
Comment (mkopec) [inline docs/transparent-validation/twonkie/introduction.md:78]: I don't think it's actually an issue with dfu-util, it's a status message from STM describing why DFU mode was entered

**Review by mkopec — CHANGES_REQUESTED** [review]

File: docs/transparent-validation/twonkie/introduction.md (hunk 2 above)
Comment (Psotas) [reply docs/transparent-validation/twonkie/introduction.md]: Done: df0c124778f253bf37e494de16c287cba3b2d653
Replaced by:
```
    > If you see ```DFU state(10) = dfuERROR, status(10) = Device's firmware is```
    ```corrupt. It cannot return to run-time (non-DFU) operations```
    > this is a status message from STM describing why DFU mode was entered.
    > Flashing is probably working properly
```

Comment (mkopec) [reply docs/transparent-validation/twonkie/introduction.md]: Thanks

**Review by mkopec — APPROVED** [review]

**Pull request merged**